<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vice_interface</class>
 <widget class="QWidget" name="vice_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="viewWnd">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>可视化效果展示</string>
   </property>
  </widget>
  <widget class="QToolButton" name="book_Img">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>171</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/img/book1.png</normaloff>:/img/book1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>170</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="dow_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>收藏</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>shuxing</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="bname_lab">
      <property name="text">
       <string>书名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="bauthor_lab">
      <property name="text">
       <string>作者：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ISBN_lab">
      <property name="text">
       <string>ISBN:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>741</width>
     <height>301</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:15px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:24px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; background-color:#ffffff;&quot;&gt;简介：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Header"/>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1001</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="toolLayout"/>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>211</width>
     <height>541</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>书库</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
